CenteredCollectionView/Classes/CenteredCollectionViewFlowLayout.swift

```diff
@@ -166,6 +182,8 @@ private extension CenteredCollectionViewFlowLayout {
 			proposedCenterOffset = proposedContentOffset.x + collectionView.bounds.size.width / 2
 		case .vertical:
 			proposedCenterOffset = proposedContentOffset.y + collectionView.bounds.size.height / 2
+        default:
+            proposedCenterOffset = .zero
```

Comment:
The spacing here doesn't match the rest of the code. Would you mind making the stacking consistent with the rest of the file?